src/views/layouts/organisms/Profile.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image";
+import { memo } from "react";
+import profile from "@public/icons/profile.png";
+
+export default memo(function Profile() {
+  return (
+    <section className="flex items-center gap-2 pt-4 pb-6 sm:gap-3">
+      <div className="h-[32px] w-[32px] sm:h-[64px] sm:w-[64px]">
+        <Image
+          src={profile}
+          alt="프로필"
+          width={64}
+          height={64}
+          className="h-full w-full"
```

Comment:
width, height 지정 후 css로 크기 바꿔주면 NextJS의 레이지로딩 최적화에 영향이 있을 것 같은데, 혹시 이렇게 fill속성 사용하는걸로 수정하면 원하는 동작이 안나올까요??

```suggestion
      <Image
        src={profile}
        alt="프로필"
        fill
        className="object-cover"

```

Reply:
오옹 해당 속성이 있는걸 잊고 있었어요!
해당 부분 적용하면 좋을 것 같습니당.
`fill`을 적용하려면 부모 요소에 `relative`가 적용되어야한다고하는데 여기서 바로 변경할 수 있나요??

Reply:
저도 제안기능 처음써봐요..ㅋㅋ 아무래도 들여쓰기 이슈도 있으니 직접 수정하시는게 😂

Reply:
ㅋㅋㅋㅋ넵 적용해서 커밋 추가했습니당